.github/workflows/ci_kind.yaml

```diff
@@ -0,0 +1,76 @@
+name: Helm Chart CI
+
+on:
+  - pull_request
+#  - push
+  
+# Only run the latest job
+concurrency:
+  group: '${{ github.workflow }} @ ${{ github.ref || github.run_id }}'
+  cancel-in-progress: true
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+          submodules: recursive
+        
+      - name: Set up podman
```

Comment:
isn't there another way to install Podman? eg with https://github.com/redhat-actions/podman-login ?